svcdef/consolidate_http.go

```diff
@@ -22,13 +25,13 @@ func consolidateHTTP(sd *Svcdef, protoFiles []io.Reader) error {
 		lex := svcparse.NewSvcLexer(pfile)
 		protosvc, err := svcparse.ParseService(lex)
 		if err != nil {
-			if strings.Contains("'options' or", err.Error()) {
+			if strings.Contains(err.Error(), "'option' or 'additional_bindings'") {
```

Comment:
I don't know what you mean when you say "'this' or 'that'"? There's not any special syntax at play, the error I'm trying to handle in this case has a string value of:

```
parser expected 'option' or 'additional_bindings' while parsing options in line '20', instead found '}'
```

Also, I'm implementing the `errors.Cause()` behavior because that would be awesome!